cli/aws_orbit/cleanup.py

```diff
@@ -168,3 +169,41 @@ def delete_cert_from_iam(context: "FoundationContext") -> None:
             pass
         else:
             raise ex
+
+
+def delete_kubeflow_roles(env_stack_name: str, region: str) -> None:
+    iam_client = boto3_client("iam")
+
+    roles = iam_client.list_roles()
+
+    regex_comp = re.compile(rf"kf-.*-{region}-{env_stack_name}")
+
+    for role in roles.get("Roles"):
+        role_name = role.get("RoleName")
+
+        if regex_comp.fullmatch(role_name):
+            _logger.info(f"Removing role {role_name} - checking for attached policies")
+            role_policies = iam_client.list_role_policies(RoleName=role_name).get("PolicyNames")
```

Comment:
what if these policies are AWS managed policies ? I think we should find if there are true kf-* policies and only delete those.

Reply:
I modified the for loop to check policies that start with kf-